### ISSUE TITLE
Timed out waiting for DB

### ISSUE THREAD

**Issue by jiivan** [golemfactory/yagna#1848]
https://discord.com/channels/687954211702439971/837225385779134495/943906160241692683

```[2022-02-17T16:12:55.197+0000 INFO yapapi.summary] [Job 1] Terminated agreement with wopr
[2022-02-17T16:12:56.261+0000 INFO yapapi.summary] [Job 1] Accepted invoice from 'wopr', amount: 0.000571956777222222
[2022-02-17T16:17:40.710+0000 INFO yapapi.summary] Received proposals from 595 providers so far
[2022-02-17T16:25:12.207+0000 ERROR yapapi.summary] [Job 1] Computation failed, reason: (500)
Reason: Internal Server Error
HTTP response headers: <CIMultiDictProxy('Content-Length': '77', 'Content-Type': 'application/json', 'Date': 'Thu, 17 Feb 2022 16:24:46 GMT')>
HTTP response body: {"message":"Failed to get events from DB: timed out waiting for connection."}
```

**Comment by etam** [comment]
I feel this happens more frequently recently

**Comment by nieznanysprawiciel** [comment]
It is possible that problem will disappear after fixes to file descriptors in `v0.10`.
Let's wait to see if problem will still exist

**Comment by etam** [comment]
I haven't seen this in a while and the graph look much better now
![Screenshot_20220426_164839](https://user-images.githubusercontent.com/220752/165327897-6c363d1c-6045-4a16-924b-81bb2f433998.png)
